<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>631</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <kerning>false</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>FoodTech</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../food.png</normaloff>../food.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>661</width>
      <height>621</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Copperplate Gothic Bold</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(170, 0, 127);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Inserir</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>281</width>
        <height>90</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>Ambiente</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ambiente">
         <property name="styleSheet">
          <string>color:rgb(0, 0, 0)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>Atendimento</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="atendimento">
         <property name="styleSheet">
          <string>color:rgb(0, 0, 0)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="cadastrar">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>310</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>9</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Cadastrar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>340</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>experiências</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>DADOS</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tabela">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>370</y>
        <width>411</width>
        <height>181</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 9pt &quot;Copperplate Gothic Bold&quot;;
border-color: rgb(255, 85, 0);
color: rgb(255, 85, 0);</string>
      </property>
      <column>
       <property name="text">
        <string>Tipo</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Restaurante</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Prato</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Nota</string>
       </property>
      </column>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>180</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>14</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Avaliação (0-100)</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>591</width>
        <height>145</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>Restaurante</string>
         </property>
         <property name="textFormat">
          <enum>Qt::AutoText</enum>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="restaurante">
         <property name="styleSheet">
          <string>color:rgb(0, 0, 0)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>Prato</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="prato">
         <property name="styleSheet">
          <string>color:rgb(0, 0, 0)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>Tipo de Culinária</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="tipo">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
         <item>
          <property name="text">
           <string>Escolher...</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Brasileira</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Mexicana</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Japonesa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Chinesa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Grega</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Indiana</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Espanhola</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Francesa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Italiana</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Americana (EUA)</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Tailandesa</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Peruana</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>24</x>
        <y>210</y>
        <width>281</width>
        <height>90</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>Comida</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="comida">
         <property name="styleSheet">
          <string>color:rgb(0, 0, 0)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>Preço</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="preco">
         <property name="styleSheet">
          <string>color:rgb(0, 0, 0)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>350</y>
        <width>146</width>
        <height>221</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QPushButton" name="ordenar_restaurante">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Copperplate Gothic Bold&quot;;
background-color: rgb(255, 85, 0);</string>
         </property>
         <property name="text">
          <string>Ordenar por Nome</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ordenar_tipo">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Copperplate Gothic Bold&quot;;
background-color: rgb(255, 85, 0);</string>
         </property>
         <property name="text">
          <string>Ordenar por tipo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ordenar_nota">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
           <underline>false</underline>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Copperplate Gothic Bold&quot;;
background-color: rgb(255, 85, 0);</string>
         </property>
         <property name="text">
          <string>Ordenar por nota</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="limpar">
         <property name="font">
          <font>
           <family>Copperplate Gothic Bold</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Copperplate Gothic Bold&quot;;
color: rgb(255, 255, 255);
background-color: rgb(170, 0, 127);</string>
         </property>
         <property name="text">
          <string>LIMPAR TABELA</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Estatísticas</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="title">
       <string>Melhor Experiência</string>
      </property>
      <widget class="QLabel" name="label_11">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 127);</string>
       </property>
       <property name="text">
        <string>Restaurante:</string>
       </property>
      </widget>
      <widget class="QLabel" name="melhor_restaurante">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>33</y>
         <width>141</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_15">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 127);</string>
       </property>
       <property name="text">
        <string>Prato:</string>
       </property>
      </widget>
      <widget class="QLabel" name="melhor_prato">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>80</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>391</width>
        <height>121</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Copperplate Gothic Bold</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="title">
       <string>Pior Experiência</string>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 127);</string>
       </property>
       <property name="text">
        <string>Restaurante:</string>
       </property>
      </widget>
      <widget class="QLabel" name="pior_restaurante">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label_17">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 127);</string>
       </property>
       <property name="text">
        <string>Prato:</string>
       </property>
      </widget>
      <widget class="QLabel" name="pior_prato">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>80</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Copperplate Gothic Bold</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(170, 0, 127);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="atualizar_estatisticas">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>480</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Copperplate Gothic Bold&quot;;
background-color: rgb(154, 0, 115);
color: rgb(255, 255, 255);
background-color: rgb(170, 0, 127);</string>
      </property>
      <property name="text">
       <string>Atualizar Estatísticas</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="styleSheet">
    <string notr="true">COLOR: rgb(255, 85, 0);</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>680</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuaRQUIVO">
    <property name="title">
     <string>Arquivo</string>
    </property>
    <addaction name="salvar"/>
    <addaction name="abrir"/>
   </widget>
   <addaction name="menuaRQUIVO"/>
  </widget>
  <action name="salvar">
   <property name="text">
    <string>Salvar Arquivo</string>
   </property>
   <property name="toolTip">
    <string>Salvar Arquivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="abrir">
   <property name="text">
    <string>Abrir Arquivo</string>
   </property>
   <property name="toolTip">
    <string>Abrir Arquivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
